mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -145,6 +146,7 @@ HMEC:
 		Cost: 3000
 	Tooltip:
 		Name: Mammoth Mk. II
+		Description: Slow heavy walker.\n  Strong vs Infantry, Vehicles, Aircraft and Buildings\n  Weak vs Nothing\nMaximum 1 can be build.
```

Comment:
"armed with dual railguns and rocket launchers".

Also you can add something like "GDI's ultimate war machine", if you can make it sound good ;)